apps/prairielearn/src/pages/administratorJobSequence/administratorJobSequence.html.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from 'zod';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function administratorJobSequence({
+  job_sequence,
+  resLocals,
+}: {
+  job_sequence: any;
```

Comment:
As discussed on Zoom: we don't yet have a good type for this object, so we're leaving it as `any` for now. This will improve once we add types to `lib/server-jobs-legacy`.